src/web/command_control/client/components/Settings.ts

```diff
@@ -95,6 +95,11 @@ export let GlobalSettings = {
         drift_time: 10
     }),
 
+    //Default depth parameter for echo task
+    listenParameters: Load<ListenParameters>('echoParameters', {
```

Comment:
What is the difference between echoParameters and listenParameters? If they are the same let's be consistent with the naming...if they are different, a comment explaining the difference could be helpful

Reply:
updated - they were the same. Matt changed his mind about and decided the feature should be called "Listen" instead of "Echo" while I was still finishing up the code so I must have missed this instance.